src/Tizen.Applications.WatchfaceComplication/Tizen.Applications.WatchfaceComplication.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
If you want to include this module in Tizen.NET package, the following information is unnecessary.